dataeng/jobs/analytics/RetirementJobEdxTriggers.groovy

```diff
@@ -0,0 +1,145 @@
+                downstreamJobName: 'retirement-partner-report-cleanup',
+                triggerJobNamePrefix: 'retirement-partner-report-cleanup-trigger',
+                environmentDeployment: 'prod-edge',
+                extraMembersCanBuild: [],
+                cron: '30 7 * * *',  // 08:30 UTC every day.
+                disabled: false
+            ]
+        ]
+
+        jobConfigs.each { jobConfig ->
+
+            // ########### user-retirement-trigger-<environment> ###########
+            // This defines the job which triggers the collector job for a given environment.
+            dslFactory.job("${jobConfig.triggerJobNamePrefix}-${jobConfig.environmentDeployment}") {
+                description("Scheduled trigger of the ${jobConfig.downstreamJobName} job for the ${jobConfig.environmentDeployment} environment")
+
+                disabled(jobConfig.disabled)
+
+                // Only a subset of edx employees should be allowed to control this job,
+                // but customer support can read and discover.
+                authorization {
+                    blocksInheritance(true)
+                    List membersWithFullControl = ['edx*edx-data-engineering', 'edx*testeng', 'edx*devops']
+                    membersWithFullControl.each { emp ->
+                        permissionAll(emp)
+                    }
+                    jobConfig.extraMembersCanBuild.each { emp ->
+                        permission('hudson.model.Item.Read', emp)
+                        permission('hudson.model.Item.Discover', emp)
+                        permission('hudson.model.Item.Build', emp)
+                        permission('hudson.model.Item.Cancel', emp)
+                    }
+                    // TODO PLAT-2036: uncomment the following two lines when we add the
+                    // appropriate github group.
+                    //permission('hudson.model.Item.Read', 'edx/customer-support')
+                    //permission('hudson.model.Item.Discover', 'edx/customer-support')
+                }
+
+                // Disallow this job to have simultaneous instances building at the same
+                // time.  This might help prevent race conditions related to triggering
+                // multiple retirement driver jobs against the same user.
+                concurrentBuild(false)
+
+                triggers {
+                    // Build every hour.
+                    cron(jobConfig.cron)
+                }
+
+                // keep jobs around for 30 days
+                // allVars contains the value for DAYS_TO_KEEP_BUILD which will be used inside AnalyticsConstants.common_log_rotator
+                logRotator common_log_rotator(allVars)
+
+                wrappers {
+                    // DENG-1063/PSRE-1204: Temporarily avoid using the build-name-setter
+                    // plugin while it's currently broken due to plugin incompatibilities.
+                    //buildName('#${BUILD_NUMBER}')
```

Comment:
This comment no longer applies because we're moving to a new jenkins instance, so I think we can re-enable buildName.

Reply:
Alright sure, I have removed these lines